src/dbt_client/dbtCloudIntegration.ts

```diff
@@ -199,8 +213,18 @@ export class DBTCloudProjectIntegration
     } catch (error) {
       // TODO: telemetry + better error
       window.showErrorMessage(
-        "Error occurred while importing validate_sql: " + error,
+        "Error occurred while initializing Python environment: " + error,
+      );
+    }
+    // TODO: find better way to share this logic between dbt detection and execution
```

Comment:
you could create a service or simple utility to share this duplicated logic

Reply:
Extracted to function